arkouda/numeric.py

```diff
@@ -2141,3 +2123,68 @@ def clip(
     if hi is not None:
         pda1 = where(pda1 > hi, hi, pda1)
     return pda1
+
+
+def median(pda):
+    """
+    Compute the median of a given array.  1d case only, for now.
+
+    Parameters
+    ----------
+    pda: pdarray
+        the input data, in pdarray form, numeric type
+
+    Returns
+    -------
+    an np.float64 value, which is the median of the entire pdarray
+
+    Examples
+    --------
```

Comment:
Would you please write the examples as executable code, something like the following?

```python
    >>> import arkouda as ak
    >>> arkouda.connect()
    >>> pda = ak.array([0,4,7,8,1,3,5,2,-1])
    >>> ak.sort(pda)
    array([-1 0 1 2 3 4 5 7 8])

    >>> ak.median(pda)
    3

```

Reply:
Done.